styles.css

```diff
@@ -200,12 +201,18 @@ hr.component__separator {
 .code-display iframe{
     width: 100%;
     border: none;
+    border:0;
```

Comment:
line 203 has border:none & you've given border:0 right after it. Please look into it unless it has been done intentionally.

Reply:
I did this by mistake. I will remove `border: 0;`. Thank you for pointing it out.